cfgov/cfgov/settings/base.py

```diff
@@ -376,7 +376,10 @@
 
 # Email
 ADMINS = admin_emails(os.environ.get('ADMIN_EMAILS'))
-EMAIL_SUBJECT_PREFIX = os.environ.get('EMAIL_SUBJECT_PREFIX')
+
+if DEPLOY_ENVIRONMENT:
+    EMAIL_SUBJECT_PREFIX = u'[{}] '.format(DEPLOY_ENVIRONMENT)
+
```

Comment:
is there supposed to be an `else` here to set it to a default, or is that happening somewhere else?